module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,30 @@
+#include "compression.hpp"
+#include <algorithm>
+#include <iostream>
+
+CompressedBitmap compressGrayscale(const Bitmap& bitmap) {
+    CompressedBitmap compressedBitmap;
+    compressedBitmap.reserve(height * width);
+    for (const auto& row : bitmap) {
```

Comment:
You can use std::transform to avoid using for loop or std::for_each.

Reply:
`for (const auto& row : bitmap)` for me is equal to `std::for_each` :)